crates/configuration/src/version3/metadata/mutations.rs

```diff
@@ -8,5 +8,4 @@ use serde::{Deserialize, Serialize};
 #[serde(rename_all = "camelCase")]
 pub enum MutationsVersion {
     V1,
-    VeryExperimentalWip,
```

Comment:
I was about to say "won't this break for some people" then remembered that's why we named it like this.